common/component_interface_utils/include/component_interface_utils/rclcpp/service_client.hpp

```diff
@@ -40,10 +41,30 @@ class Client
     client_ = client;  // to keep the reference count
   }
 
+  /// Send request.
+  typename WrapType::SharedResponse call(
+    const typename WrapType::SharedRequest request, double timeout = 0.0)
+  {
+    if (!client_->service_is_ready()) {
+      RCLCPP_INFO_STREAM(logger_, "client unready: " << SpecT::name);
+      throw ServiceUnready(SpecT::name);
+    }
+
+    const auto future = this->async_send_request(request);
+    if (timeout != 0.0) {
```

Comment:
[IMO] Using optional can make it more descriptive?

```suggestion
    const typename WrapType::SharedRequest request, std::optional<double> timeout = {})
  {
    if (!client_->service_is_ready()) {
      RCLCPP_INFO_STREAM(logger_, "client unready: " << SpecT::name);
      throw ServiceUnready(SpecT::name);
    }

    const auto future = this->async_send_request(request);
    if (timeout) {
```

Reply:
Done https://github.com/autowarefoundation/autoware.universe/pull/1429/commits/e2076f6187edf47a9a58210bac5cd9ba41c1c808, https://github.com/autowarefoundation/autoware.universe/pull/1429/commits/031c59a8b208304f27db3a595fb3be402c49abda.